packages/eventual-send/package.json

```
@@ -28,6 +28,7 @@
   "devDependencies": {
     "@agoric/assert": "^0.2.2",
     "@agoric/install-ses": "^0.5.2",
+    "@agoric/ses-ava": "^0.1.0+1-dev",
```

ses-ava is coming from npm, which does not carry our dev versions.
```suggestion
    "@agoric/ses-ava": "^0.1.0",
```